index.html

```diff
@@ -1,102 +1,99 @@
-              fill="currentColor"
-            />
-          </svg>
-          Read me</a
-        >
-      </h3>
-    </footer>
-  </body>
-</html>
+    }
+  </style>
+  <link rel="stylesheet" href="style.css" media="print" onload="this.media='all'" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@300;800&display=swap" rel="stylesheet" />
+</head>
+
+<body>
+  <header>
+    <h1>Fidaa Bashir</h1>
+    <nav>
+      <ul>
+        <li><a href="#about">About Me</a></li>
+        <li><a href="#projects">Projects</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main tabindex="0">
+    <section id="about">
+      <header>
+        <h2>About Me</h2>
+      </header>
+      <p>
+       A CYF Software development trainee. A mum for 2 lovely boys. To get into Tech was one of my dreams that I'm trying to
+      achieve, and I'm working hard to fulfill this dream, so I'm so grateful for Code Your Future for giving me this great
+      opportunity to be one of CYF's Trainees. In addition to that, I also have a deep experience in business administration.
+
+      </p>
+    </section>
+    <section id="projects">
+      <header>
+        <h2>Projects Showcase</h2>
+      </header>
+      <p>
+      Here are some of the projects, that I have done during my training with CYF so far, you can also explore my Github and
+      Netlify for more projects.
+      </p>
+      <div class="cards">
+        <a class="card" href="https://fidaa-bashir-website.netlify.app/" target="_blank">
+          <img alt="Personal website" src="https://i.postimg.cc/05hyn94x/Screenshot-2023-10-03-at-22-20-13.png">
+          <span>Personal Website</span>
+        </a>
+        <a class="card" href="https://deploy-preview-475--cyf-bfr.netlify.app/" target="_blank">
+        <img alt="Bikes for Refugees" src="https://i.postimg.cc/zvYXJjKv/Screenshot-2023-10-03-at-22-31-20.png">
+        <span>Bikes for Refugees</span>
+    </section>
+    <section id="contact">
+      <header>
+        <h2>Contact me</h2>
+      </header>
+      <p>
+        You can contact me here:-
+      </p>
+      <ul>
+        <li><a id="profile-link" target="_blank" href="https://github.com/Fidaabashir89"><i class="github"></i>Github</a></li>
```

Comment:
I love the fancy animations on these cards. However, you do need to have links to both the github and the deployed project as stated in the readme, not just the deplyed project